src/screens/CreatePost/elements/CreatePollContainer/CreatePollContainer.android.js

```diff
@@ -21,7 +21,8 @@ function CreatePollContainer({
   selectedtime = {day: 1, hour: 0, minute: 0},
   ontimechanged = () => {},
   polls,
-  expiredobject = {day: 7, hour: 24}
+  expiredobject = {day: 7, hour: 24},
+  isAnonym
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `isAnonym` has been added to the component. Ensure that this prop is always passed when using this component, otherwise it may cause unexpected behavior or crashes due to undefined values.
```diff
-  expiredobject = {day: 7, hour: 24}
+  expiredobject = {day: 7, hour: 24},
+  isAnonym
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->